templates/tuned.conf.erb

```diff
@@ -1,5 +1,5 @@
 [main]
-include=virtual-guest
+include=<%= scope.lookupvar('::tuned_base_profile') %>
```

Comment:
I think this should be better implemented as `<%= @tuned_base_profile %>`.  See doco [here](https://docs.puppet.com/puppet/latest/reference/lang_template_erb.html#variable).